01/jcastle13/wordvalue.py

```diff
@@ -0,0 +1,68 @@
+from data import DICTIONARY, LETTER_SCORES
+
+def load_words():
+    """Load dictionary into a list and return list"""
+    #print("load_words method here")
+    file = open("dictionary.txt", "r")
+    temp_list = []
+    for i in file:
+        temp = str(i)
+        temp_list.append(i.replace('\n',''))
+    file.close()
+    return(temp_list)
+    pass
+
+def calc_word_value(word=""):
+    """Calculate the value of the word entered into function
+    using imported constant mapping LETTER_SCORES"""
+    #print("calc_word_value method here")
+    #print("word to calculate:", word)
+
+    sum = 0
```

Comment:
can use sum() builtin in one line but more importantly if you do it this way use sum_ to distinguish from the builtin